kubetest/extract_k8s.go

```diff
@@ -74,7 +74,7 @@ func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
 		`^(bazel)$`: localBazel,
 		`^(local)`:  local,
-		`^gke-?(default|channel-(rapid|regular|stable)|latest(-\d+.\d+)?)?$`: gke,
+		`^gke-?(default|channel-(rapid|regular|stable)|latest(-\d+.\d+(.\d(-gke)?)?)?)?$`: gke,
```

Comment:
I think the patch version should also be `\d+`

Reply:
done, and added a test for it